content/en/examples/secret/ssh-auth-secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret-ssh-auth
+type: kubernetes.io/ssh-auth
+data:
+  # the data is abbreviated in this example
+  ssh-privatekey: |
+    MIIEpQIBAAKCAQEAulqb/Y ...
```

Comment:
Is this manifest `apply`-able? If not, I'd prefer to change this so that the data are at least base64 decodable (even if not an actual private key)

Reply:
Updated with a base64 encoded string